org.musicbrainz.Picard.json

```diff
@@ -66,35 +162,26 @@
                 "--disable-nonfree",
                 "--enable-optimizations",
                 "--enable-pthreads",
-
                 "--disable-bzlib",
                 "--disable-libgsm",
                 "--enable-libvorbis",
                 "--enable-zlib",
                 "--disable-libxcb",
                 "--disable-lzma",
-
                 "--disable-programs",
                 "--disable-network",
-
                 "--disable-protocols",
                 "--enable-protocol=file",
-
                 "--disable-devices",
-
                 "--disable-muxers",
-
                 "--disable-demuxers",
                 "--enable-demuxer=flac",
                 "--enable-demuxer=mp3",
                 "--enable-demuxer=ogg",
-
                 "--disable-parsers",
                 "--enable-parser=flac",
                 "--enable-parser=vorbis",
-
                 "--disable-encoders",
-
```

Comment:
I had added those empty lines on purpose, to visually separate the various "sections" of ffmpeg options.

If they are causing problems and need to be removed, I'd like that to happen in its own commit with an explanation. Otherwise, let's keep them please. :slightly_smiling_face: